### ISSUE TITLE
Library filtering

### ISSUE THREAD

**Issue by XakepSDK** [wvengen/proguard-maven-plugin#91]
No way to filter library inclusion. You can filter injars, but not libraryjars.

**Comment by lasselindqvist** [comment]
True, but the libraryjars are explicitly given as a list (https://github.com/wvengen/proguard-maven-plugin/blob/master/src/main/java/com/github/wvengen/maven/proguard/ProGuardMojo.java#L154)
Could you describe a situation where an exclusion filter is needed on top of that?

**Comment by XakepSDK** [comment]
Multi release jar with java 9+ classes in `META-INF/versions` directory

**Comment by Marcono1234** [comment]
The comments of #103 say that it addressed this issue here. Should it be closed then?
